src/collectd.conf.pod

```diff
@@ -2343,6 +2343,69 @@ Enabled by default, collects unknown (and thus presented as numeric only) query
 
 =back
 
+=head2 Plugin C<dpdkstat>
+
+The I<dpdkstat plugin> collects information about DPDK interfaces using the
+extended NIC stats API in DPDK.
+
+B<Synopsis:>
+
+ <Plugin "dpdkstat">
+    Interval 1
+    Coremask "0x4"
+    MemoryChannels "4"
+    ProcessType "secondary"
+    FilePrefix "rte"
+    EnabledPortMask 0xffff
+    PortName "interface1"
+    PortName "interface2"
+ </Plugin>
+
+B<Options:>
+
+=over 4
+
+=item B<Interval> I<seconds>
```

Comment:
Many plugins used to implement an `Interval` option and eventually we implemented this generic syntax:

```
<LoadPlugin "dpdkstat">
  Interval 2.0
</LoadPlugin>
```

Plugins read the value configured this way by calling `plugin_get_interval()`, which gets the value from thread-local storage. Would that be an option for this plugin as well?